lib/chat_models/chat_bumblebee.ex

```diff
@@ -123,7 +126,8 @@ defmodule LangChain.ChatModels.ChatBumblebee do
     # # more focused and deterministic.
     # field :temperature, :float, default: 1.0
 
-    field :template_format, Ecto.Enum, values: [:inst, :im_start, :zephyr, :llama_2, :llama_3]
+    field :template_format, Ecto.Enum,
+      values: [:inst, :im_start, :zephyr, :llama_2, :llama_3, :llama_3_1_json_tool_calling, :llama_3_1_custom_tool_calling, :llama_3_2_custom_tool_calling]
```

Comment:
This is the line where I think the merge conflict is. It should be a very simple fix though.